src/components/carbon-provider/carbon-provider.component.js

```diff
@@ -1,15 +1,40 @@
-import React from "react";
+import React, { createContext, useContext } from "react";
 import PropTypes from "prop-types";
 import { ThemeProvider } from "styled-components";
 
 import mintTheme from "../../style/themes/mint";
 
-const CarbonProvider = ({ children, theme = mintTheme }) => (
-  <ThemeProvider theme={theme}>{children}</ThemeProvider>
-);
+export const NewValidationContext = createContext({
+  validationRedesignOptIn: false,
+});
+
+const CarbonProvider = ({
+  children,
+  validationRedesignOptIn,
```

Comment:
```suggestion
  validationRedesignOptIn = false,
```

Reply:
and it should be the last parameter with being the new one

Reply:
It's already defaulted to `false` in the `createContext`, I'll move it to the last parameter

Reply:
If you do what I said below (this bit `<NewValidationContext.Provider value={{ validationRedesignOptIn }}>`) - then it will change it to undefined unless you set the default as `false` here